sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -29,7 +29,7 @@ npm install @azure/ai-form-recognizer@4.0.0-beta.3
 ## Getting Started
 
 ```javascript
-import { DocumentAnalysisClient } from "@azure/ai-form-recognizer";
+const { DocumentAnalysisClient } = require("@azure/ai-form-recognizer");
 import { DefaultAzureCredential } from "@azure/identity";
```

Comment:
Should we change the other import to require as well?

Reply:
Just noticed this as well and pushed an update.